ci_support/run_docker_build.sh

```diff
@@ -40,12 +40,9 @@ echo "$config" > ~/.condarc
 conda clean --lock
 
 conda update --yes --all
-conda install --yes conda-build==1.18.2
+conda install --yes conda-build
 conda info
 
-# Temporary fix to install opengl drivers
-yum install -y mesa-libGL-devel
```

Comment:
We decided we would include the same libraries that manylinux includes. This was one of the libraries that was acceptable according to the [listing in the PEP]( https://www.python.org/dev/peps/pep-0513/#the-manylinux1-policy ). Thus we add it to the docker image in this PR ( https://github.com/pelson/Obvious-CI/pull/63 ). It is also in our current image after it got moved over [here]( https://github.com/conda-forge/docker-images/blob/fbde090bd608caa720d5caad861aa382a8bf3f5c/linux-anvil/Dockerfile#L28 ). So, it is ok to lose this line.